### PR TITLE
Adding docstring to tf.keras.activations.deserialize

### PR DESCRIPTION
**Pull request by eito-fis**
Redoing [this PR](https://github.com/tensorflow/tensorflow/pull/29282) against master instead of 2.0.

### CODE REVIEW

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -292,6 +292,16 @@ def serialize(activation):
 
 @keras_export('keras.activations.deserialize')
 def deserialize(name, custom_objects=None):
+  """Inverse of the `serialize` function.
+  Arguments:
+      config: Optimizer configuration dictionary.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: don't think so?

File: tensorflow/python/keras/activations.py [hunk 2]
```diff
@@ -292,6 +292,16 @@ def serialize(activation):
...
+      custom_objects: Optional dictionary mapping
+          names (strings) to custom objects
+          (classes and functions)
+          to be considered during deserialization.
+  Returns:
+      A Keras Optimizer instance.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: don't think so?

**Comment by gbaned** [comment]
@eito-fis Could you please address the reviewer comments. Thanks!

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
